src/Controller/Api/IdeasWorkshop/IdeaExtendController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace AppBundle\Controller\Api\IdeasWorkshop;
+
+use AppBundle\Entity\IdeasWorkshop\Idea;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+class IdeaExtendController
```

Comment:
It will be better to transform `IdeaPublishController` to `IdeaController` and to have one controller per entity than one controller par entity action.